Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -402,73 +428,87 @@ - (void)startObservingLookupDismissalIfNeeded
 
 - (void)_windowDidOrderOnScreen:(NSNotification *)notification
 {
-    _impl->windowDidOrderOnScreen();
+    if (_impl)
+        _impl->windowDidOrderOnScreen();
 }
 
 - (void)_windowDidOrderOffScreen:(NSNotification *)notification
 {
-    _impl->windowDidOrderOffScreen();
+    if (_impl)
+        _impl->windowDidOrderOffScreen();
 }
 
 - (void)_windowDidBecomeKey:(NSNotification *)notification
 {
-    _impl->windowDidBecomeKey([notification object]);
+    if (_impl)
+        _impl->windowDidBecomeKey([notification object]);
 }
 
 - (void)_windowDidResignKey:(NSNotification *)notification
 {
-    _impl->windowDidResignKey([notification object]);
+    if (_impl)
+        _impl->windowDidResignKey([notification object]);
 }
 
 - (void)_windowDidMiniaturize:(NSNotification *)notification
 {
-    _impl->windowDidMiniaturize();
+    if (_impl)
+        _impl->windowDidMiniaturize();
 }
 
 - (void)_windowDidDeminiaturize:(NSNotification *)notification
 {
-    _impl->windowDidDeminiaturize();
+    if (_impl)
+        _impl->windowDidDeminiaturize();
 }
 
 - (void)_windowDidMove:(NSNotification *)notification
 {
-    _impl->windowDidMove();
+    if (_impl)
+        _impl->windowDidMove();
 }
 
 - (void)_windowDidResize:(NSNotification *)notification
 {
-    _impl->windowDidResize();
+    if (_impl)
+        _impl->windowDidResize();
 }
 
 - (void)_windowWillBeginSheet:(NSNotification *)notification
 {
-    _impl->windowWillBeginSheet();
+    if (_impl)
+        _impl->windowWillBeginSheet();
 }
 
 - (void)_windowDidChangeBackingProperties:(NSNotification *)notification
 {
     CGFloat oldBackingScaleFactor = [[notification.userInfo objectForKey:NSBackingPropertyOldScaleFactorKey] doubleValue];
-    _impl->windowDidChangeBackingProperties(oldBackingScaleFactor);
+    if (_impl)
+        _impl->windowDidChangeBackingProperties(oldBackingScaleFactor);
```

Comment:
We probably should add early return (`if (!_impl)`) before `CGFloat oldBackingScaleFactor =...`

Reply:
Fixed.